client/boot/polyfills.js

```diff
@@ -1,24 +1,23 @@
 /**
  * External dependencies
  */
-import '@babel/polyfill';
+import 'core-js/stable';
+import 'regenerator-runtime/runtime';
```

Comment:
Note: we need to depend on this directly now, as it used to be included by `@babel/polyfill`, but it's not in `core-js/stable`. See https://github.com/zloirock/core-js/blob/master/docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md#babelpolyfill